src/content/app-platform/apps/observability/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+linkTitle: Observability Stack
+title: Managed Observability Stack (Monitoring, Logging, Service Mesh)
```

Comment:
```suggestion
linkTitle: Observability stack
title: Managed observability stack (monitoring, logging, service mesh)
```

FYI: We decided for sentence casing instead of title case.